src/components/CategoryCard/CategoryCard.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react'
+import { Header } from 'decentraland-ui'
+import { Props } from './CategoryCard.types'
+import './CategoryCard.css'
+
+export default class CategoryCard extends React.PureComponent<Props> {
+  handleClick = () => {
+    const { category, onClick } = this.props
+    onClick(category.name)
+  }
+  render() {
+    const { category } = this.props
+
+    if (!category || category.assets.length === 0) return null
+
+    const { name } = category
+    const asset = category.assets.find(asset => !!asset.main) || category.assets[0]
```

Comment:
Do you think it would be a good idea to create a method for `category.assets.find(asset => !!asset.main) || category.assets[0]`? We could keep it on category/utils so we don't leak how we select "main" assets